float64 dxPixel
float64 dyPixel
